# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/WebIdentityChild.cpp
# Commit: 63d9972bab1f
# Full Hash: 63d9972bab1f267a940ec30c7cea7a4c41e54fa6
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-06-06 08:52:03
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing leaks @ mochitest.toml
#   
#   This reverts commit a97cb1d6c08bc38e8644bfc916298b847dbac3a1.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 801e25cb4bfd -r 63d9972bab1f dom/credentialmanagement/identity/WebIdentityChild.cpp
--- a/dom/credentialmanagement/identity/WebIdentityChild.cpp	Thu Jun 05 18:51:36 2025 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,23 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=8 sts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "mozilla/dom/WebIdentityChild.h"
-#include "mozilla/dom/WebIdentityHandler.h"
-
-namespace mozilla::dom {
-
-void WebIdentityChild::ActorDestroy(ActorDestroyReason why) {
-  if (mHandler) {
-    mHandler->ActorDestroyed();
-    mHandler = nullptr;
-  }
-}
-
-void WebIdentityChild::SetHandler(WebIdentityHandler* aHandler) {
-  mHandler = aHandler;
-}
-
-}  // namespace mozilla::dom